Jenkinsfile

```diff
@@ -105,6 +105,7 @@ pipeline {
                         }
                     }
                     steps {
+                        sh "rm iof.conf"
```

Comment:
This is exactly what the `clean:` argument of `sconsBuild` below is for.
```suggestion
```

But frankly I have been more and more of the opinion that this should be baked into [`sconsBuild`](https://github.com/daos-stack/pipeline-lib/blob/master/vars/sconsBuild.groovy)'s built-in list of [cleaning targets](https://github.com/daos-stack/pipeline-lib/blob/b1ecd6150d74519b5e6d5d3a1caaa9553915af12/vars/sconsBuild.groovy#L131) in [pipeline-lib](https://github.com/daos-stack/pipeline-lib).